src/components/views/elements/AccessibleButton.tsx

```diff
@@ -75,7 +75,7 @@ type IProps<T extends keyof JSX.IntrinsicElements> = DynamicHtmlElementProps<T>
     disabled?: boolean;
     className?: string;
     triggerOnMouseDown?: boolean;
-    onClick: ((e: ButtonEvent) => void | Promise<void>) | null;
+    onClick?: ((e: ButtonEvent) => void | Promise<void>) | null;
```

Comment:
I think null here might be intentional for the caller to demonstrate intent

Reply:
Makes sense, removed this change.